tools/occlum

```diff
@@ -129,20 +129,43 @@ cmd_init() {
     mkdir -p image
     mkdir -p image/bin
     mkdir -p image/lib
+    mkdir -p image/lib64
     mkdir -p image/root
     mkdir -p image/host
     mkdir -p image/tmp
+    local occlum_glibc_lib=/usr/local/occlum-glibc/lib
```

Comment:
I am surprised that this is not under `/usr/local/occlum` and that we are having a new folder under `/usr/local/`.

The original reason of installing the gcc-musl toolchain at `/usr/local` is to follow the convention of the installation path for toolchains. And also that the headers and libraries of the toolchain should be visible to users. In contrast, all contents in `/opt/occlum` is not supposed to be accessed directly by users.

I think the new glibc is more akin to the contents in `/opt/occlum`, which are internal to Occlum. So the new glibc should be installed in `/opt/occlum`. Make sense?

Reply:
Right